ee/codegen/src/__test__/helpers/node-data-factories.ts

```diff
@@ -1423,6 +1426,7 @@ export function genericNodeFactory(
         },
       },
     ],
+    adornments: adornments ?? undefined,
```

Comment:
nit: I think this is equivalent

```suggestion
    adornments,
```